stylesheets/index.css

```diff
@@ -0,0 +1,71 @@
+@import url("https://fonts.googleapis.com/css2?family=PT+Sans+Narrow:wght@400;700&display=swap");
+
+.img-container {
+	margin: auto;
+	width: 95%;
+	display: flex;
+}
+
+.img-container .homepage-img {
+	width: 100%;
+	border-radius: 4px;
+}
+
+.image-text-overlay {
+	position: absolute;
+	left: 20rem;
+	top: 25rem;
+	display: flex;
+	flex-direction: column;
+	align-items: center;
+}
+
+.image-text-overlay .img-text {
+	font-family: "PT Sans Narrow", sans-serif;
+	font-size: 3rem;
+	text-align: center;
+	color: white;
```

Comment:
use variable

Reply:
alright , got it , will do the necassary changes in the next commit